.editorconfig

```diff
@@ -0,0 +1,209 @@
+# top-most EditorConfig file
+root = true
+
+[*]
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+# See https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/code-style-rule-options?view=vs-2019  for supported options
+[*.cs]
+indent_size = 4
+indent_style = tab
+
+dotnet_analyzer_diagnostic.category-Globalization.severity = none
+
+
+# Code Analysis tweaks for style rules
+dotnet_diagnostic.IDE0008.severity = none
```

Comment:
i just copied all of these from a work project so if the styles are annoying we can nuke them.